Explorer modified with latest java client sdk 2.0.4 (#17)
scalar-labs/scalardl-tools: explorer/README.md

Scalar DL Explorer has four commands:

> - `explorer initialize`: used to register your certificate and contracts(GetContract, ScanContract). You have to execute this command one time only.

Reviewer: ```suggestion
- `explorer initialize`: used to register your certificate and contracts(GetContract, ScanContract). This command needs to be executed only one time.
```

Author: Thank you for review change.
Review changes updated in 135ba8e5358cdc9bfd02084406fd65ba68b3ddd4
